Themes/default/css/jquery.sceditor.css

```diff
@@ -197,24 +198,24 @@ div.sceditor-dropdown div {
 }
 .sceditor-container iframe,
 .sceditor-container textarea {
-	display: block;
-	-ms-flex: 1 1 0%;
-	flex: 1 1 0%;
 	line-height: normal;
 	border: 0;
 	outline: none;
 	font-size: 14px;
 	color: #111;
 	box-sizing: border-box;
-	padding: 5px;
+	padding: 0 5px;
 	margin: 0px;
 	resize: none;
 	background: #fff;
-	display: block;
+	display: flex;
+	flex-basis: 175px;
+	flex-grow: 1;
 	min-width: 100%;
 	max-width: 100%;
-	height: auto !important;
+	height: 175px;
```

Comment:
Now what would happen if a textedit control/mod specifies a longer height? Should this then be  a custom data attribute?

Reply:
Can your JavaScript code target this using height instead of minHeight?

Reply:
Is there an example of this behavior also?? To look into it